### PR TITLE
Introduce fabric app

### PR DESCRIPTION
**Pull request by yurikoles**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/fabric.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'fabric' do
+  version :latest
+  sha256 :no_check
+
+  # ssl-download-crashlytics-com.s3.amazonaws.com was verified as official when first introduced to the cask
```
Comment (reitermarkus) [inline Casks/fabric.rb]: The URL section in the comment is not unique to the vendor.

Comment (yurikoles) [inline Casks/fabric.rb]: why? it contains `ssl-download-crashlytics-com`

Comment (reitermarkus) [inline Casks/fabric.rb]: My bad, didn't know Crashlytics is now part of Fabric.

**Comment by reitermarkus** [comment]
Please delete the parts of the pull request template that are not relevant and fill out the rest.

**Comment by adidalal** [comment]
The download link is walled, couldn't verify.

**Comment by vitorgalvao** [comment]
> The download link is walled, couldn't verify.

Yep. https://github.com/caskroom/homebrew-cask/pull/16503 and https://github.com/caskroom/homebrew-cask/pull/17319.

**Pull request closed without merging**